pkg/core/native/crypto_test.go

```diff
@@ -29,6 +29,80 @@ func TestSha256(t *testing.T) {
 		require.Equal(t, "47dc540c94ceb704a23875c11273e16bb0b8a87aed84de911f2133568115f254", hex.EncodeToString(c.sha256(ic, []stackitem.Item{stackitem.NewByteArray([]byte{1, 0})}).Value().([]byte)))
 	})
 }
+func TestKeccak256(t *testing.T) {
```

Comment:
s/TestKeccak256/TestKeccak256_Compat
And add a corresponding comment with the link to the source of compat examples.

Reply:
Link to C# examples is needed.